src/js/components/Toolbar/__tests__/__snapshots__/Toolbar-test.js.snap

```diff
@@ -165,8 +165,8 @@ exports[`DataFilters renders 1`] = `
 .c13 {
   margin-left: 0px;
   margin-right: 0px;
-  margin-top: 6px;
-  margin-bottom: 6px;
+  margin-top: 12px;
```

Comment:
Changes only due to jsdom, does not appear in browser. Correct value is re-assigned below